kolibri/core/deviceadmin/utils.py

```diff
@@ -41,6 +41,19 @@ def default_backup_folder():
     return os.path.join(KOLIBRI_HOME, "backups")
 
 
+def get_backup_files():
+    """
+    Returns all backups from current kolibri version.
+    """
+    default_path = default_backup_folder()
+    backups = os.listdir(default_path)
+    prefix = "db-v{}"
```

Comment:
We're not looking for any specific version so we can drop the string format here.
```suggestion
    prefix = "db-v"
```
I think this is the reason that the test is failing, because it's effectively filtering out all the DB backups, because none of them start with the string literal `db-v{}`.